src/Groups/gsets.jl

```diff
@@ -760,6 +761,38 @@ all_blocks(G::PermGroup) = Vector{Vector{Int}}(GAP.Globals.AllBlocks(G.X))
 #TODO: Do we really want to act on the set of moved points?
 
 
+"""
+    rank_action(G::PermGroup, L::AbstractVector{Int} = 1:degree(G))
+
+Return the rank of the action of `G` on `L`, i.e., the number of orbits
+of the point stabilizer.
```

Comment:
... acting on what? I would have assumed after reading this that the point being stabilized is not included, but in your code it is... so the rank is 1 iff the group has degree 1 (and thus is trivial), and 2 iff the group acts 2-transitively?